### ISSUE TITLE
Incorrect Gaussian state

### ISSUE THREAD

**Issue by uday-sankar** [qojulia/QuantumOptics.jl#345]
I wanted to create a gaussian packet of particular energy so I tried making Gaussian packets with different momenta fixing the sigma. The energy of the wavepacket increases when I increase the po_x value but after a point, the energy sharply decreases even with a higher p0_x.  And one more question, is h-bar=1 in the software? i.e, all the units are in hartree atomic units or S.I units?

**Comment by Lightup1** [comment]
As for the unit of $\hbar$, you can check #294.
As for the energy of Gaussian packets, I think the dependance could be natural since the min or max momentum of the  momentum space is dependent on the position space.

**Comment by uday-sankar** [comment]
Yeah, thanks for your response. Later going through some theory I understood the problem. And thank you for clarifying the hbar unit.
